src/lib.rs

```diff
@@ -54,12 +54,11 @@ pub fn run_server(build_args: Option<BuildArgs>) {
     let ip = config.ip.clone();
     let port = config.port;
 
-    let server = server::Instance::new(config, build_args);
-
     println!("server running on http://{}:{}", ip, port);
-    Server::http(&*format!("{}:{}", ip, port))
+    let addr = format!("{}:{}", ip, port).parse().unwrap();
+    Http::new().bind(&addr, move || Ok(server::Instance::new(config.clone(), build_args.clone())))
```

Comment:
You shouldn't need to clone `config` and `build_args` - they aren't used later, so should be OK to move.

Reply:
if I don't clone i get: ``` this closure implements `FnOnce`, not `Fn` ``` 
and if I don't move i get: 
 ``` closure may outlive the current function, but it borrows `config`, which is owned by the current function ```